Doc/whatsnew/3.9.rst

```diff
@@ -600,7 +600,8 @@ Removed
 
 * Methods ``getchildren()`` and ``getiterator()`` in the
   :mod:`~xml.etree.ElementTree` module have been removed.  They were
-  deprecated in Python 3.2.  Use functions :func:`list` and :func:`iter`
+  deprecated in Python 3.2.  Use functions :func:`list` and
+  :meth:`Element.iter()<xml.etree.ElementTree.Element.iter>`
   instead.  The ``xml.etree.cElementTree`` module has been removed.
```

Comment:
```suggestion
* Methods ``getchildren()`` and ``getiterator()`` of classes
  :class:`~xml.etree.ElementTree.ElementTree` and
  :class:`~xml.etree.ElementTree.Element` in the :mod:`~xml.etree.ElementTree`
  module have been removed.  They were deprecated in Python 3.2.
  Use ``iter(x)`` or ``list(x)`` instead of ``x.getchildren()`` and
  ``x.iter()`` or ``list(x.iter())`` instead of ``x.getiterator()``.
  The ``xml.etree.cElementTree`` module has been removed.
```

Reply:
Thank you for the suggestion @serhiy-storchaka .
I have included the recommended changes in my latest commit.